src/js/chapterDOM.js

```diff
@@ -1,5 +1,7 @@
 import { makeDraggable, makeDroppable } from "./drag-drop";
 
+let section;
```

Comment:
Maybe we can move this variable to the beginning of the createChapterStructure and define it right away as new section node there? It should work the same and we will avoid global variables. If it won't work, then it could stay as it is.

Reply:
If section is moved into `createChapterStructure` then you can't access section inside `setupChapterPage`.

I think global variables inside a module are alright. You can't access section variable from outside of `chapterDOM`. There is a closure. So it is not really a global variable, it is just globally available inside of `chapterDOM`